DemoApp/package.json

```diff
@@ -8,39 +8,39 @@
     "test": "jest"
   },
   "dependencies": {
-    "@react-native-community/async-storage": "^1.5.0",
-    "appcenter": "^2.5.0",
-    "appcenter-analytics": "^2.5.0",
-    "appcenter-auth": "^2.5.0",
-    "appcenter-crashes": "^2.5.0",
-    "appcenter-data": "^2.5.0",
-    "appcenter-push": "^2.5.0",
-    "metro": "^0.54.1",
-    "react": "16.8.3",
-    "react-native": "0.59.9",
+    "@react-native-community/async-storage": "1.6.2",
+    "appcenter": "2.5.0",
+    "appcenter-analytics": "2.5.0",
+    "appcenter-auth": "2.5.0",
+    "appcenter-crashes": "2.5.0",
+    "appcenter-data": "2.5.0",
+    "appcenter-push": "2.5.0",
+    "metro": "0.54.1",
+    "react": "16.11.0",
+    "react-native": "0.61.2",
```

Comment:
Revert the `react` and `react-native` version. We deliberately stay on RN 0.59.x on DemoApp, while TestApp keeps up to date to the latest RN. It's a good practice to keep `react` in sync with `react-native` version because newer `react` could break older `react-native`.